circle.yml

```diff
@@ -1,28 +0,0 @@
-machine:
-  python:
-    version: 2.7.11
-
-dependencies:
-  pre:
-    - pyenv install -s 3.5.4
-    - pyenv install -s 3.6.0
-    - pyenv global 2.7.11 3.5.4 3.6.0
-    - pip install -r requirements-dev.txt
-
-  post:
-    - make docs
-    - mv docs/build ${CIRCLE_ARTIFACTS}/docs
-    - cd ${CIRCLE_ARTIFACTS}; zip -r docs.zip docs
-
-test:
-  pre:
-    - mkdir -p ${CIRCLE_TEST_REPORTS}/junit ${CIRCLE_ARTIFACTS}/coverage
-  override:
-    - tox --
-      --verbose
-      -p no:sugar
-      --cov=koordinates
```

Comment:
can we keep the coverage report too? Can be [linked from the annotations](https://buildkite.com/docs/agent/v3/cli-annotate#embedding-and-linking-artifacts-in-annotations)

Reply:
I think that would be only marginally useful. No one will click it, and if you do then you have to click around to compare the percentages between builds.

More useful if there's a big fat change% number somewhere (preferably on github PRs saying "The coverage decreased by 10%"). I couldn't find an easy way to do that from buildkite.

I'm sure I could mess around with writing my own custom annotate, but it's more useful to post to github than to store in CI anyway. 

One option is to sign up to codecov, which is free for OSS and has an upload tool which supports buildkite, though that's not immediately obvious (see https://github.com/codecov/codecov-bash/pull/115). 

It annotates the github PR as a status check: https://docs.codecov.io/docs/commit-status